x-pack/plugins/cases/public/components/create/description.test.tsx

```diff
@@ -45,29 +46,70 @@ describe('Description', () => {
   });
 
   it('it renders', async () => {
-    const result = appMockRender.render(
+    appMockRender.render(
       <MockHookWrapperComponent>
         <Description {...defaultProps} />
       </MockHookWrapperComponent>
     );
 
-    expect(result.getByTestId('caseDescription')).toBeInTheDocument();
+    expect(screen.getByTestId('caseDescription')).toBeInTheDocument();
   });
 
   it('it changes the description', async () => {
-    const result = appMockRender.render(
+    appMockRender.render(
       <MockHookWrapperComponent>
         <Description {...defaultProps} />
       </MockHookWrapperComponent>
     );
 
+    const description = screen.getByTestId('euiMarkdownEditorTextArea');
+
     userEvent.type(
-      result.getByRole('textbox'),
+      description,
       `${specialChars.selectAll}${specialChars.delete}My new description`
     );
 
     await waitFor(() => {
       expect(globalForm.getFormData()).toEqual({ description: 'My new description' });
     });
   });
+
+  it('shows an error when description is empty', async () => {
+    appMockRender.render(
+      <MockHookWrapperComponent>
+        <Description {...defaultProps} />
+      </MockHookWrapperComponent>
+    );
+
+    const description = screen.getByTestId('euiMarkdownEditorTextArea');
+
+    userEvent.clear(description);
+    userEvent.type(description, '  ');
+
+    await waitFor(() => {
+      expect(screen.getByText('A description is required.')).toBeInTheDocument();
+    });
+  });
+
+  it('shows an error when description is too long', async () => {
+    const longDescription = Array(MAX_DESCRIPTION_LENGTH / 2 + 1)
```

Comment:
I am curious about `MAX_DESCRIPTION_LENGTH / 2 + 1` 🙂.

Reply:
yeah it's because array to string conversion adds comma after each element. 
This was not an issue on API tests, but in UI unit tests hang if I add 60K characters instead of 30K to input element 😄 

But I can change it to `'a'.repeat(MAX_DESCRIPTION_LENGTH) ` to avoid confusion

Reply:
Oh, I see. Yes, I think it is better if we do `'a'.repeat(MAX_DESCRIPTION_LENGTH)` to avoid confusion.